pkgs/by-name/ri/rime-ice/package.nix

```diff
@@ -0,0 +1,37 @@
+{
+  lib,
+  stdenvNoCC,
+  fetchFromGitHub,
+  unstableGitUpdater,
+}:
+
+stdenvNoCC.mkDerivation {
+  pname = "rime-ice";
+  version = "0-unstable-2024-09-16";
+
+  src = fetchFromGitHub {
+    owner = "iDvel";
+    repo = "rime-ice";
+    rev = "2ea99c5e4f3afaa24ebef61d17ea097557ca36b3";
+    hash = "sha256-AzH1rZFqEH8sovZZfJykvsEmCedEZWigQFHWHl6/PdE=";
```

Comment:
Could you check the hash again?
```
error: hash mismatch in fixed-output derivation '/nix/store/y283j2aa086rkyj0jf177nnhvsfl6pdn-source.drv':
         specified: sha256-AzH1rZFqEH8sovZZfJykvsEmCedEZWigQFHWHl6/PdE=
            got:    sha256-6leug6CxZjg855fjW2mKPLcmMTO/KE9xjLm9hbZH2Fo=
```

Reply:
Seems like the hash changed, I will update it.